tests/bsim/bluetooth/audio_samples/unicast_audio_client/prj.conf

```diff
@@ -1,16 +1,2 @@
-# This file content is just a copy of the unicast client sample prj.conf
-
-CONFIG_BT=y
-CONFIG_LOG=y
-CONFIG_BT_CENTRAL=y
-CONFIG_BT_AUDIO=y
-CONFIG_BT_BAP_UNICAST_CLIENT=y
-CONFIG_BT_ISO_TX_BUF_COUNT=4
-# Support an ISO channel per ASE
-CONFIG_BT_ISO_MAX_CHAN=4
-CONFIG_BT_BAP_UNICAST_CLIENT_GROUP_STREAM_COUNT=4
-CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT=2
-CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT=2
-CONFIG_BT_KEYS_OVERWRITE_OLDEST=y
-
-CONFIG_BT_EXT_ADV=y
+# Please build using the sample configuration file:
```

Comment:
why not remove the file?

Reply:
You mean this file? 

Didn't consider that TBH :D Was just following the structure by @aescolar

Reply:
Ref: https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bsim/bluetooth/audio_samples/broadcast_audio_sink/prj.conf